plugins/woocommerce-blocks/tests/e2e/tests/rating-filter/rating-filter.block_theme.spec.ts

```diff
@@ -0,0 +1,281 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+/**
+ * Internal dependencies
+ */
+import ProductCollectionPage from '../product-collection/product-collection.page';
+import { cli } from '@woocommerce/e2e-utils';
+
+const blockData = {
+	name: 'Filter by Rating',
+	slug: 'woocommerce/rating-filter',
+	urlSearchParamWhenFilterIsApplied: '?rating_filter=1',
+};
+
+const test = base.extend< {
+	productCollectionPageObject: ProductCollectionPage;
+} >( {
+	productCollectionPageObject: async (
+		{ page, admin, editor, templateApiUtils, editorUtils },
+		use
+	) => {
+		const pageObject = new ProductCollectionPage( {
+			page,
+			admin,
+			editor,
+			templateApiUtils,
+			editorUtils,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( `${ blockData.name } Block`, () => {
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( {
+			name: 'woocommerce/filter-wrapper',
+			attributes: {
+				filterType: 'rating-filter',
+				heading: 'Filter By Rating',
+			},
+		} );
+		await editor.openDocumentSettingsSidebar();
+	} );
+
+	test( "should allow changing the block's title", async ( { page } ) => {
+		const textSelector =
+			'.wp-block-woocommerce-filter-wrapper .wp-block-heading';
+
+		const title = 'New Title';
+
+		await page.fill( textSelector, title );
```

Comment:
`page.fill` [is discouraged](https://playwright.dev/docs/api/class-page#page-fill) so let's use the `locator.fill` instead.